src/makeInfinityAdder.js

```diff
@@ -5,8 +5,24 @@
  * @return {function}
  */
 
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(x) {
```

Comment:
```suggestion
function makeInfinityAdder() {
```
This function shouldn't receive any parameters. By the way, you don't even use it.
Pay attention to the example 
![image](https://github.com/mate-academy/js_task-inifinite-sum-closures/assets/96409563/0c5aa57e-9479-4186-8307-d988b92a9810)